tox.ini

```diff
@@ -8,7 +8,7 @@ deps =
     -rrequirements.txt
     pytest-cov
 passenv = TRAVIS* PG* SQLALCHEMY_DATABASE_TEST_URI PERSISTENCE_DIR FLASK_APP SECRET_KEY
-commands = py.test --cov portal --cov-report xml:"{toxinidir}/coverage.xml" []
+commands = py.test --ignore=tests/integration_tests --cov portal --cov-report xml:"{toxinidir}/coverage.xml" []
```

Comment:
I think `--ignore=tests/integration_tests` will be required for every invocation of `py.test` , if you don't want the selenium/integration tests to run.

@pbugni 
Should the integration tests run when `py.test` is called?

Reply:
@ivan-c I'm not sure how to best integrate - this is a CI issue in my mind, which I feel you're more qualified to answer.

Shall we start w/ full integration, and back off if we find it too noisy?